lib/ts/recipe/oauth2client/index.ts

```diff
@@ -14,12 +14,37 @@
  */
 
 import Recipe from "./recipe";
-import { RecipeInterface, APIInterface, APIOptions } from "./types";
+import { RecipeInterface, APIInterface, APIOptions, ProviderConfigWithOIDCInfo, OAuthTokens } from "./types";
 
 export default class Wrapper {
     static init = Recipe.init;
+
+    static async getAuthorisationRedirectURL(input: { redirectURIOnProviderDashboard: string }) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getAuthorisationRedirectURL(input);
+    }
+
+    static async exchangeAuthCodeForOAuthTokens(input: {
+        providerConfig: ProviderConfigWithOIDCInfo;
+        redirectURIInfo: {
+            redirectURIOnProviderDashboard: string;
+            redirectURIQueryParams: any;
+            pkceCodeVerifier?: string | undefined;
+        };
+    }) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.exchangeAuthCodeForOAuthTokens(input);
```

Comment:
Ideally, this would call `getProviderConfig` itself (esp. since it isn't exposed) instead of taking `providerConfig` as a param

Reply:
done